src/components/menuPage/NotFound.js

```diff
@@ -0,0 +1,16 @@
+import { useNavigate } from "react-router-dom";
+
+const NotFound = () => {
+    let navigate=useNavigate()
+    let clickHandler=()=>{
+        navigate("/menu")
+    }
+    return ( 
+        <div className="mx-3 mt-4">
+        <h1>Item Not Found!</h1>
+        <button className="btn btn-primary mt-3" onClick={clickHandler}>Return to Main Menu</button>
```

Comment:
Could we label the button as menu listings or another name that is more appropriate? "Return to Main Menu" does sound right. What do you think?

Reply:
The button color is not alignment with the site's style. Blue button color doesn't look right. How about the yellow button color shown in /contact page?

Reply:
Do you mean 'Return to Menu Listings' or just 'Menu Listings'?

Reply:
Have fixed and made a new commit